packages/dx-react-grid-bootstrap3/src/templates/virtual-table/virtual-box.jsx

```diff
@@ -29,6 +29,7 @@ function getVisibleItems(options) {
     const itemInfo = options.itemInfo(index);
     const itemSize = itemInfo.size;
     const itemStick = itemInfo.stick;
+    const key = itemInfo.key;
```

Comment:
Does eslint allow to use this construction? I mean, should we use object decomposition instead?

Reply:
There were no errors. But it looks like _eslint_ will fail after the #431 merge.